src/modules/70-pipeline/components/ArtifactsSelection/ArtifactInterface.ts

```diff
@@ -244,6 +244,25 @@ export interface JenkinsArtifactProps {
   isMultiArtifactSource?: boolean
 }
 
+export interface BambooArtifactProps {
+  key: string
+  name: string
+  expressions: string[]
+  context: number
+  initialValues: BambooArtifactType
+  handleSubmit: (data: BambooArtifactType) => void
+  artifactIdentifiers: string[]
+  isReadonly?: boolean
+  selectedArtifact: ArtifactType | null
+  allowableTypes: AllowedTypes
+  isMultiArtifactSource?: boolean
+}
+
+export interface ArtifactFormikProps {
```

Comment:
```suggestion
export interface ArtifactFormikProps<Values> {
  formik: FormikProps<Values>
  formClassName?: string
}
```

`FormikValues` is just for the values. We need all formik props here including values.